tests/fast/EncryptedBackupCorrectness.toml

```diff
@@ -1,5 +1,6 @@
 [configuration]
 tenantModes = ['required']
+encryptModes = ['domain_aware']
```

Comment:
Are there any other tests, where we need to update this config?

Reply:
Currently this test is the only one that requires encryption mode to be `domain_aware` and not chosen at random

Reply:
If there are other tests that require this we'll probably find out as we replace the knob usage and can address it then